config/initializers/job_configurations.rb

```diff
@@ -188,6 +188,11 @@
         cron: cron_24h,
         args: -> { [14.days.ago] },
       },
+      # Send emails when unused GPO verification codes have expired
+      send_gpo_code_expiration_notices: {
+        class: 'SendGpoCodeExpirationNoticesJob',
+        cron: cron_24h,
+      },
```

Comment:
Recommend deploying without the cron entry, testing in prod, and then enabling the job in a separate PR, especially since new instances will have run a migration that old instances will be missing.

Reply:
Agreed. Updated in 9fb8790b5dc3aad15f76fa1cac8cd236f6f8aa0e